roles/fio_distributed/templates/client.yaml

```diff
@@ -70,15 +72,23 @@ spec:
           name: "fio-hosts-{{ trunc_uuid }}"
           defaultMode: 0777
       restartPolicy: Never
-{% if metadata is defined and metadata.collection|default(false) is sameas true and metadata.targeted|default(true) is sameas true %}
+{% if metadata is defined and (metadata.collection|default(false)) and (metadata.targeted|default(true)) %}
       initContainers:
       - name: backpack-{{ trunc_uuid }}
         image: {{ metadata_image | default('quay.io/cloud-bulldozer/backpack:latest') }}
         command: ["/bin/sh", "-c"]
-{% if metadata is defined and metadata.force|default(false) is sameas true %}
-        args: ["python3 stockpile-wrapper.py -s {{ elasticsearch.server }} -p {{ elasticsearch.port }} -u {{ uuid }} -n $my_node_name -N $my_pod_name --redisip {{ bo.resources[0].status.podIP }} --redisport 6379 --force"]
-{% else %}
-        args: ["python3 stockpile-wrapper.py -s {{ elasticsearch.server }} -p {{ elasticsearch.port }} -u {{ uuid }} -n $my_node_name -N $my_pod_name --redisip {{ bo.resources[0].status.podIP }} --redisport 6379"]
+        args:
```

Comment:
@mulbc I like this layout much better than the single line approach that was in there prior. That said it runs into the same errors as above and does not actually execute as is. However, changing it to this did work:

```
        args:
          - "python3
             stockpile-wrapper.py
             -s {{ elasticsearch.server }}
             -p {{ elasticsearch.port }}
             -u {{ uuid }}
             -n $my_node_name
             -N $my_pod_name
             --redisip {{ bo.resources[0].status.podIP }}
             --redisport 6379
{% if metadata is defined and metadata.force|default(false) %}
             --force
{% endif %}
             "
```